### PR TITLE
John conroy/saved list fixes

### LINKED ISSUE #1526: Revise design of "Edit Saved Status" when no other lists exist
I got this after clicking on "Edit Saved Status" after saving my very first data set:

image

There should be a message about what people should do (e.g. navigate to "My Lists") and create a list in order to assign the current dataset to another list.

### LINKED ISSUE #1527: Change text for list save confirmation
Change text (without the emphasis) to "... All **lists** are currently stored ..." ("data" is a bit confusing in this context).

image

Should be easy to fix.

### LINKED ISSUE #1529: Fix order of column labels in list item table
The entity type and group column headers need to be swapped as they don't match the column contents:

image

### PR DESCRIPTION
**Pull request by john-conroy**
In addition to showing "You don't have any lists. To assign this [dataset/sample/donor] to one or more lists, create a list in My Lists.", I disabled the save button.

Fix #1526.

Screen Shot 2021-02-11 at 9 44 17 AM

Fix #1527.

Screen Shot 2021-02-11 at 9 48 57 AM

Screen Shot 2021-02-11 at 9 49 06 AM

 Fix #1529.

Screen Shot 2021-02-11 at 9 47 44 AM

### CODE REVIEW

File: context/app/static/js/components/savedLists/EditSavedStatusDialog/EditSavedStatusDialog.jsx [hunk 1]
```diff
@@ -46,22 +47,30 @@ function EditSavedStatusDialog({ dialogIsOpen, setDialogIsOpen, uuid }) {
...
-        />
+        savedListsExist ? (
+          <AddToList
+            selectedLists={selectedLists}
+            addToSelectedLists={addToSelectedLists}
+            removeFromSelectedLists={removeFromSelectedLists}
+          />
+        ) : (
+          <Typography>
+            {`You don't have any lists. To assign this ${entity_type.toLowerCase()} to one or more lists, create a list in My Lists.`}
```
Comment (ngehlenborg) [inline context/app/static/js/components/savedLists/EditSavedStatusDialog/EditSavedStatusDialog.jsx:64]: Is it possible to make "My Lists" a link to the ".../my-lists" URL?

**Review by ngehlenborg — CHANGES_REQUESTED** [review]
One comment that might not be possible to fix. The disabled Save button is a great idea!

File: context/app/static/js/components/savedLists/EditSavedStatusDialog/EditSavedStatusDialog.jsx (hunk 1 above)
Comment (john-conroy) [reply context/app/static/js/components/savedLists/EditSavedStatusDialog/EditSavedStatusDialog.jsx]: Screen Shot 2021-02-11 at 11 43 01 AM

**Review by ngehlenborg — APPROVED** [review]

**Comment by mccalluc** [comment]
I think john's last commit also fixes #1536. Linked.

**Review by mccalluc — APPROVED** [review]

**Comment by mccalluc** [comment]
(I thought this would also address something I noticed in #1536, but it doesn't, but that's ok: Not a high priority. Sorry for the noise.)

**Comment by john-conroy** [comment]
@tsliaw could you make a design for what @mccalluc is describing in #1536. Since an entity can be linked to multiple lists, I'm not sure how that should look. Also a list can contain an entity which is no longer saved.

**Review by tsliaw — APPROVED** [review]
These changes look good. As for #1536, I'll have to think about to do that so feel free to merge these changes and I'll make the mockup to fix the link to the lists in #1536.

**Review by ngehlenborg — APPROVED** [review]

**Pull request merged**